src/components/LinterMessage/index.spec.tsx

```diff
@@ -144,6 +161,32 @@ describe(__filename, () => {
     expect(root).toHaveClassName(`.${styles.inline}`);
   });
 
+  it('can be marked as highlighted', () => {
+    const uid = 'some-uid';
+    const location = createFakeLocation({
+      search: queryString.stringify({ [messageUidQueryParam]: uid }),
```

Comment:
side note: it's really cool that you can link someone directly to a linter message.